### PR TITLE
[TFAC-437] Adding code to Complete a Mission

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: graphql/database/missions/__tests__/utils.test.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ const mockDefaultMissionReturn = {
   tabCount: 258,
   squadMembers: [
     {
+      userId: 'cL5KcFKHd9fEU5C9Vstj3g4JAc73',
```
Comment (amaliwanag2) [inline graphql/database/missions/__tests__/utils.test.js:146]: this is mocking the return value we return to the client, where we hide userIds.  It looks like you're no longer using this value either

File: graphql/database/missions/completeMission.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  const mostDayTabsMember = mission.squadMembers.reduce((prev, current) =>
+    prev.missionMaxTabsDay > current.missionMaxTabsDay ? prev : current
+  )
+  const mostTotalTabsMember = mission.squadMembers.reduce((prev, current) =>
+    prev.tabs > current.tabs ? prev : current
+  )
+
+  const endOfMissionAwards = [
+    {
+      awardType: 'Consistent Kitty',
```
Comment (amaliwanag2) [inline graphql/database/missions/completeMission.js:40]: lets make these constants

**Review by amaliwanag2 — CHANGES_REQUESTED** [review]
Just one small change.  Otherwise, Looks good to me!

File: graphql/database/missions/__tests__/utils.test.js (hunk 1 above)
Comment (jedtan) [reply graphql/database/missions/__tests__/utils.test.js:146]: I'm not sure I follow - I did amend `getCurrentUserMission` but it sounds like I shouldn't do that? I was adding it so I would have the userIds of squad members when calculating end of mission awards.

Comment (amaliwanag2) [reply graphql/database/missions/__tests__/utils.test.js:146]: actually that's' fine, graphql will still mask the user ids on calls to the FE so disregard

**Review by amaliwanag2 — APPROVED** [review]
LGTM

**Pull request merged**